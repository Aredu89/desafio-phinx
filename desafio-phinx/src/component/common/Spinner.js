import React from 'react';

const Spinner = () => {
  return(
    <div className="d-flex justify-content-center w-100 mb-3">
      <div className="spinner-border" role="status">
        <span className="sr-only">Loading...</span>
      </div>
    </div>
  )
}

export default Spinner;