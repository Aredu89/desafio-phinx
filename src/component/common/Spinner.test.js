import React from 'react';
import { shallow } from 'enzyme';
import Spinner from './Spinner.js';

describe('Spinner',()=>{
  it('Render sin props',()=>{
    const component = shallow(<Spinner />);
    expect(component).toMatchSnapshot()
  })
})