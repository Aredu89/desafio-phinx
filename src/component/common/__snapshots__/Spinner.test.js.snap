// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Render sin props 1`] = `
<div
  className="d-flex justify-content-center w-100 mb-3"
>
  <div
    className="spinner-border"
    role="status"
  >
    <span
      className="sr-only"
    >
      Loading...
    </span>
  </div>
</div>
`;
